HPC/scripts-setup/0-install-acfl.sh

```diff
@@ -4,7 +4,7 @@
 # install acfl will include armpl automatically
 mkdir -p /shared/tools
 cd /shared/tools
-wget -O arm-compiler-for-linux_23.04_Ubuntu-20.04_aarch64.tar 'https://developer.arm.com/-/media/Files/downloads/hpc/arm-compiler-for-linux/23-04/arm-compiler-for-linux_23.04_Ubuntu-20.04_aarch64.tar?rev=5f0c6e9758aa4409ab9e6a3891c791a4&revision=5f0c6e97-58aa-4409-ab9e-6a3891c791a4'
-tar xf arm-compiler-for-linux_23.04_Ubuntu-20.04_aarch64.tar
-./arm-compiler-for-linux_23.04_Ubuntu-20.04/arm-compiler-for-linux_23.04_Ubuntu-20.04.sh \
+wget -O arm-compiler-for-linux_23.04.1_Ubuntu-20.04_aarch64.tar 'https://developer.arm.com/-/media/Files/downloads/hpc/arm-compiler-for-linux/23-04-1/arm-compiler-for-linux_23.04.1_Ubuntu-20.04_aarch64.tar?rev=52971e8fa8a8498c834e48776dfd1ca5&revision=52971e8f-a8a8-498c-834e-48776dfd1ca5'
```

Comment:
Can you make this a generic statement (not specify the hashes as it is bound to change with newer releases)

Reply:
Both the link and the hashes are going to change in newer releases.  

I am not sure about what do you mean by not specify the hashes because that's part of the address to download the library. If I remove the hashes, I get a 404 message because the link doesn't exit.

$ wget -O arm-compiler-for-linux_23.04_Ubuntu-20.04_aarch64.tar 'https://developer.arm.com/-/media/Files/downloads/hpc/arm-compiler-for-linux/23-04/arm-compiler-for-linux_23.04_Ubuntu-20.04_aarch64.tar'
HTTP request sent, awaiting response... 404 The specified blob does not exist.
2023-07-17 19:39:03 ERROR 404: The specified blob does not exist..

Reply:
What I meant was if we make it a place holder like <executable name> or just add a comment above the lines in the script to indicate that this changes and user has to pick the one from the webpage accordingly.

If hashes change over time, user will get 404 anyway...